src/main/java/nova/core/recipes/crafting/ShapedCraftingRecipe.java

```diff
@@ -255,7 +256,7 @@ public void consumeItems(CraftingGrid craftingGrid) {
 
 		for (int i = 0; i < ingredients.length; i++) {
 			Item original = mapping.Items[i];
-			Item consumed = ingredients[i].consumeOnCrafting(original, craftingGrid);
+			Item consumed = Objects.requireNonNull(ingredients[i].consumeOnCrafting(original, craftingGrid), "The result of 'ItemIngredient.consumeOnCrafting' can't be null");
```

Comment:
Use `Objects.requireNonNull` on a seperate line after the definition so the line doesn't get too long